RelationalAI/TransactionAsyncResult.cs

```diff
@@ -16,25 +16,25 @@
 namespace RelationalAI
 {
     using System.Collections.Generic;
-    using Newtonsoft.Json;
+    using Relationalai.Protocol;
 
     public class TransactionAsyncResult : Entity
     {
         public TransactionAsyncCompactResponse Transaction { get; set; }
         public List<ArrowRelation> Results { get; set; }
-        public List<TransactionAsyncMetadataResponse> Metadata { get; set; }
+        public MetadataInfo Metadata { get; set; }
         public List<object> Problems { get; set; }
```

Comment:
Let's follow this up with a PR similar to https://github.com/RelationalAI/rai-sdk-java/pull/16, introducing `GotCompleteResult`